atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -35,6 +39,9 @@
 WORKSPACE_KEY_BUILD_INFO = 'build_info'
 WORKSPACE_KEY_UPLOAD_DIR = 'koji_upload_dir'
 WORKSPACE_KEY_OVERRIDE_KWARGS = 'override_kwargs'
+FIND_CLUSTER_RETRY_DELAY = 1.0
+FAILURE_RETRY_DELAY = 1.0
+MAX_CLUSTER_FAILS = 2
```

Comment:
Is this value realistic? I'd suggest 15, so that `MAX_CLUSTER_FAILS * FAILURE_RETRY_DELAY` takes us to 15 minutes. @lcarva, agree?